app/helpers/filter_dropdown_helper.rb

```diff
@@ -97,8 +106,13 @@ def dropdown_toggler_options
       aria: { expanded: 'false', haspopup: 'true' }.merge(toggle_drop) }
   end
 
-  def translate_value(filter_attribute, t_scope)
+  def translate_value(filter_attribute, q_filter)
     return t('all') if filter_attribute.blank?
-    t(filter_attribute, t_scope)
+    q_filter = q_filter.is_a?(Symbol) ? q_filter.to_s : q_filter.first.to_s
+    if q_filter.slice! '_true'
+      t_attr(q_filter)
+    elsif q_filter.slice! '_eq'
+      t("simple_form.options.#{controller_name.singularize}.#{q_filter}.#{filter_attribute}")
+    end
```

Comment:
See https://github.com/panter/aoz-003/pull/241#discussion_r141595300

Reply:
So your refusing my PR because of this and don't review anything else in it?

Reply:
Sorry I had a meeting, didn't get around to do a full review yet :) I see it's closed now?

Reply:
@toupeira sorry, i was getting frustrated about the thing. PO expected it to be finished long time ago, and as It was getting endless, seeing it going even more endless kinda gave me a ... well. whatever

Reply:
But well, anyhow. I cannot do a refactoring respecting the other branch within this branch, and it should definiteley not be a acceptance criteria to this PR to have that filter thing refactored for it